operator/src/test/resources/custom-ingress-installed-manually.yml

```diff
@@ -0,0 +1,41 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
I think we don't need anything as complex as this. A new ingress could be created programatically, maybe even as simple as this:
```java
new IngressBuilder().withNewMetadata().withName("example-kc-ingress").endMetadata().build();
```

Reply:
Done!

Thanks.